cmd/controller/main.go

```diff
@@ -43,6 +50,19 @@ func main() {
 	lo.Must0(op.AddHealthzCheck("cloud-provider", awsCloudProvider.LivenessProbe))
 	cloudProvider := metrics.Decorate(awsCloudProvider)
 
+	serverVersion, err := op.VersionProvider.Get(ctx)
```

Comment:
```suggestion
	serverVersion := lo.Must(op.VersionProvider.Get(ctx))
```

Force getting the server version to succeed?